alfred-telemetry-solr/build.gradle

```diff
@@ -0,0 +1,73 @@
+plugins {
+    id 'eu.xenit.docker' version '5.0.7' apply false
+    id 'java'
+    id 'idea'
+}
+
+subprojects {
+    repositories {
+	mavenCentral()
+	jcenter()
+	
+	maven {
+            url 'https://artifacts.alfresco.com/nexus/content/groups/private'
```

Comment:
Is all of solr not in the public alfresco repo ? :-/

Reply:
Before alfresco 6 not all solr code is public. (e.g. alfesco-solr4:5.1.5). But I believe a public community version is compatible.

Reply:
Removed dependency on enterprise repos.